### PR TITLE
docs: add nuxt-link defaults config in nuxt.config

### PR DESCRIPTION
**Pull request by xjccc**
### 🔗 Linked issue

none

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Set `nuxt-link` defaults by `nuxt.config` is one way, and its not in the docs.

However its will be changed in the future.

So, using `defineNuxtLink` is the reccomended way?

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have added tests (if possible).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/3.api/1.components/4.nuxt-link.md [hunk 1]
```diff
@@ -88,6 +88,33 @@ Defaults can be overwritten, see [overwriting defaults](#overwriting-defaults) i
...
+export default defineNuxtConfig({
+  experimental: {
+    defaults: {
+      nuxtLink: {
+        componentName?: string;
+        externalRelAttribute?: string;
+        activeClass?: string;
+        exactActiveClass?: string;
+        prefetchedClass?: string;
+        trailingSlash?: 'append' | 'remove'
```
Comment (danielroe) [inline docs/3.api/1.components/4.nuxt-link.md:109]:
```suggestion
        // default values
        componentName: 'NuxtLink',
        externalRelAttribute: 'noopener noreferrer',
        activeClass: 'router-link-active',
        exactActiveClass: 'router-link-exact-active',
        prefetchedClass: undefined, // can be any valid string class name
        trailingSlash: undefined // can be 'append' or 'remove'
```

File: docs/3.api/1.components/4.nuxt-link.md [hunk 2]
```diff
@@ -88,6 +88,33 @@ Defaults can be overwritten, see [overwriting defaults](#overwriting-defaults) i
 
 ## Overwriting Defaults
 
+### By Nuxt Config
+
+You can overwrite `<NuxtLink>` defaults by [`nuxt.config`](https://nuxt.com/docs/api/nuxt-config#defaults)
```
Comment (danielroe) [inline docs/3.api/1.components/4.nuxt-link.md:93]:
```suggestion
You can overwrite some `<NuxtLink>` defaults in your [`nuxt.config`](https://nuxt.com/docs/api/nuxt-config#defaults)
```

File: docs/3.api/1.components/4.nuxt-link.md [hunk 3]
```diff
@@ -88,6 +88,33 @@ Defaults can be overwritten, see [overwriting defaults](#overwriting-defaults) i
 
 ## Overwriting Defaults
 
+### By Nuxt Config
```
Comment (danielroe) [inline docs/3.api/1.components/4.nuxt-link.md:91]:
```suggestion
### In Nuxt Config
```

File: docs/3.api/1.components/4.nuxt-link.md [hunk 4]
```diff
@@ -88,6 +88,33 @@ Defaults can be overwritten, see [overwriting defaults](#overwriting-defaults) i
...
+        exactActiveClass?: string;
+        prefetchedClass?: string;
+        trailingSlash?: 'append' | 'remove'
+      }
+    }
+  }
+})
+```
+
+### Recommended
```
Comment (danielroe) [inline docs/3.api/1.components/4.nuxt-link.md:116]:
```suggestion
### Custom Link Component
```

**Review by danielroe — COMMENTED** [review]
Thank you ❤️

**Review by danielroe — APPROVED** [review]

**Comment by xjccc** [comment]
> Thank you ❤️

Thank you for your hard work.❤️

**Pull request merged**